database/src/main/kotlin/no/nav/su/se/bakover/database/vedtak/VedtakPostgresRepo.kt

```diff
@@ -189,6 +211,8 @@ internal class VedtakPostgresRepo(
                         saksbehandler = saksbehandler,
                         attestant = attestant,
                         periode = periode,
+                        // TODO fjern henting fra behandling etter migrering
+                        avslagsgrunner = avslagsgrunner ?: behandling.avslagsgrunner,
```

Comment:
Dere har jo lagt ved en migrering?

Reply:
Det trengs til selve migreringen. Migreringen er todelt, 1 legger til kolonne + 1 migrerer eksisterende verdier til kolonne, sistnevnte henter ut alle objektene det skal migreres verdier for først (i det migreringen kjøres vil det ikke eksistere noen verdi på vedtaket for dette feltet)